src/Symfony/Component/VarDumper/Dumper/CliDumper.php

```diff
@@ -88,22 +88,18 @@ public function __construct($output = null, string $charset = null, int $flags =
 
     /**
      * Enables/disables colored output.
-     *
-     * @param bool $colors
      */
-    public function setColors($colors)
+    public function setColors(?bool $colors)
     {
-        $this->colors = (bool) $colors;
```

Comment:
the type cast should be kept suppose

Reply:
I'll give it a try. But maybe we should rather deprecate passing `null` to that method? That execution path had been undocumented anyway.

Reply:
Got it. I was simply fooled by a incomplete mock again. 😄 

`$colors` isn't nullable anymore, the offending test has been adjusted to produce a proper mock.